decidim-bulletin_board-app/app/javascript/packs/sandbox/index.js

```diff
@@ -0,0 +1,37 @@
+import $ from "jquery";
+
+$(async () => {
+  // UI Elements
+  const $showInputButton = $(".show-input-button");
+  const $generateVotesButton = $(".generate-votes-button");
+
+  $showInputButton.on("click", (event) => {
+    $(event.target).hide();
+    $(event.target)
+      .siblings(".generate-votes-input-section")
+      .css("display", "inline");
+  });
+
+  $generateVotesButton.on("click", (event) => {
+    const electionId = $(event.target).closest(".election").data("id");
+    const votesToGenerate = $(event.target)
+      .siblings(".generate-votes-input")
+      .val();
+    $(event.target).html(`Generating ${votesToGenerate} votes...`);
+    $(event.target).prop("disabled", true);
+
+    $.ajax({
+      url: `/sandbox/elections/${electionId}/generate_bulk_votes`,
+      method: "POST",
+      contentType: "application/json",
+      data: JSON.stringify({
+        number_of_votes: votesToGenerate,
+      }), // eslint-disable-line camelcase
+      headers: {
+        "X-CSRF-Token": $("meta[name=csrf-token]").attr("content"),
+      },
+    })
+      .done(() => $(event.target).html(`${votesToGenerate} votes generated!`))
+      .fail(() => $(event.target).html(`Failed, retry!`));
```

Comment:
Since `$.ajax` returns a `Promise` you could also write this using `async/await` like this:

```js
$generateVotesButton.on("click", async (event) => {
  // ...
  try {
    await $.ajax(...);
    $(event.target).html(`${votesToGenerate} votes generated!`);
  } catch {
    $(event.target).html(`Failed, retry!`);
  }
});
```

I am ok with both ways :)